components/admin_console/permission_schemes_settings/permission_system_scheme_settings/permission_system_scheme_settings.jsx

```diff
@@ -218,7 +218,7 @@ export default class PermissionSystemSchemeSettings extends React.Component {
     }
 
     render = () => {
-        const hasCustomSchemes = this.props.license.CustomPermissionsSchemes === 'true';
+        const hasCustomSchemes = 'true';
```

Comment:
Just curious if this is correct.  If yes, then defining `hasCustomSchemes` variable might not be necessary.

Reply:
You're right. I must have been experimenting and forgot to remove.